object_tracker/config/run.yaml

```diff
@@ -0,0 +1,6 @@
+# @package _global_
+
+defaults:
+  - _self_
+  - object_tracker: object_tracker.yaml
+  - match: match.yaml
```

Comment:
Should not be in this branch, along with the rest of the functionality we have decided that won't enter the upcoming version. 
Please move it into a different branch accordingly.